src/OrchardCore/OrchardCore.Navigation.Core/NavigationHelper.cs

```diff
@@ -11,6 +10,9 @@ namespace OrchardCore.Navigation
 {
     public class NavigationHelper
     {
+        public static bool IsAdminMenu(string name)
```

Comment:
While it's trivial, also document it like the other methods here.

Reply:
done